src/main/java/com/epam/ta/reportportal/database/dao/ProjectRepositoryCustom.java

```diff
@@ -114,4 +114,12 @@
 
 	Optional<String> findPersonalProjectName(String user);
 
+	/**
+	 * Changes status of project logs indexing for analyzer
+	 *
+	 * @param projectName Project name
+	 * @param value       Status of indexing
+	 */
+	void changeProjectIndexingStatus(String projectName, boolean value);
```

Comment:
what do it mean? enableIndexing?

Reply:
status of indexing. is it in progress or not